### PR TITLE
Fantasy Module Parser Alpha Version

### PR DESCRIPTION
**Pull request by EngineerSuite2**
- User-created Categories functional
- User-created Languages functional
- Multiple NPC Modules working

### CODE REVIEW

File: FantasyModuleParser/Main/Models/CategoryModel.cs [hunk 1]
```diff
@@ -11,5 +11,10 @@ public class CategoryModel
     {
         public string Name { get; set; }
         public List<NPCModel> NPCModels { get; set; } = new List<NPCModel>();
+
+        public string ToString()
```
Comment (lorddusk) [inline FantasyModuleParser/Main/Models/CategoryModel.cs:15]: Not to happy that a ToString is overriden to just return the Name.

CategoryModel.Name does **LITERALLY** the same, as it's both a getter and a setter.

File: FantasyModuleParser/Main/Views/FMPConfigurationView.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    /// </summary>
+    public partial class FMPConfigurationView : Window
+    {
+        private FMPConfigurationViewModel configurationViewModel;
+        public FMPConfigurationView()
+        {
+            InitializeComponent();
+
+            // Enable it so the popup window can close on the Escape key
+            this.PreviewKeyDown += (s, e) => { if (e.Key == Key.Escape) Close(); };
```
Comment (lorddusk) [inline FantasyModuleParser/Main/Views/FMPConfigurationView.xaml.cs:29]: s > sender
e > event

File: FantasyModuleParser/Main/Views/ProjectManagement.xaml.cs [hunk 3]
```diff
@@ -14,6 +15,10 @@ public partial class ProjectManagement : Window
         public ProjectManagement()
         {
             InitializeComponent();
+
+            // Enable it so the popup window can close on the Escape key
+            this.PreviewKeyDown += (s, e) => { if (e.Key == Key.Escape) Close(); };
```
Comment (lorddusk) [inline FantasyModuleParser/Main/Views/ProjectManagement.xaml.cs:20]: s > sender
e > event

File: FantasyModuleParser/Main/Models/CategoryModel.cs (hunk 1 above)
Comment (EngineerSuite2) [reply FantasyModuleParser/Main/Models/CategoryModel.cs]: Removed public string ToString()

File: FantasyModuleParser/Main/Views/ProjectManagement.xaml.cs (hunk 3 above)
Comment (EngineerSuite2) [reply FantasyModuleParser/Main/Views/ProjectManagement.xaml.cs:20]: this.PreviewKeyDown += (sender, eventArgs) => { if eventArgs.Key == Key.Escape) Close(); };

**Pull request merged**